### ISSUE TITLE
dudas arrays

### ISSUE THREAD

**Issue by javierbarbe** [javierbarbe/repo1#4]
Ivan no entiendo bien porqué en este ejercicio
repo1\ejercicioLibro200\Ej1 (metodo ej4) && repo1\ejercicioLibro200\PruebaArrayy ( metodo main)

coge cada uno de los valores introducidos... no deberia quedarse con el ultimo introducido y hacer con ese ya el cuadrado y el cubo? 

El enunciado es el siguiente

> _Define tres arrays de 20 números enteros cada una, con nombres numero,
> cuadrado y cubo. Carga el array numero con valores aleatorios entre 0 y 100. En el
> array cuadrado se deben almacenar los cuadrados de los valores que hay en el
> array numero. En el array cubo se deben almacenar los cubos de los valores que
> hay en numero. A continuación, muestra el contenido de los tres arrays dispuesto
> en tres columnas._

**Comment by ivanjimenez** [comment]
No, el ejercicio te dice que hagas 3 arrays, numero, que lleva los aleatorios, cuadrados, que lleva los cuadrados de número y cubos que lleva los cubos. Los muestras por columnas. Cada fila de las mismas lleva la posición i de cada array
